test/common_constants.go

```diff
@@ -10,26 +10,44 @@
 //
 // SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
 
-package agent
+package test
 
 import (
 	"time"
 
 	"github.com/eclipse-kanto/update-manager/api/types"
 )
 
-const testActivityID = "testActivityId"
-const interval = 1 * time.Second
+const TestActivityID = "testActivityId"
```

Comment:
probably `ActivityID` would be better here because when used it becomes `test.Test...` which is kind of redundant